view/theme/frio/theme.php

```diff
@@ -16,7 +16,8 @@
 use Friendica\Core\PConfig;
 use Friendica\Core\System;
 use Friendica\Database\DBA;
-use Friendica\Model\Profile;
+use Friendica\Model;
+use Friendica\Module\Contact;
```

Comment:
Although this is correct, please write `use Friendica\Module;` and later `Module\Contact::getContactTemplateVars` to prevent any immediate confusion.